### ISSUE TITLE
Can't Trade btc/usdt or usdt/btc - Errors

### ISSUE THREAD

**Issue by focamonca** [freqtrade/freqtrade#4174]
I added them to the whitelist on Config file but the only thing I can get is a bung of errors. 
I'd like to trade using usdt or busd but I can't and I can't understand is it's an error or what. 
Thanks in advance.

**Comment by zn3x** [comment]
What error are you exactly getting? What exchange are you using?
Please follow how to make an issue

**Comment by focamonca** [comment]
I'm using binance in dry_run
This is the error I'm getting "freqtrade | freqtrade.exceptions.OperationalException: Stake-currency 'BTC' not compatible with pair-whitelist. Please remove the following pairs: ['BTC/BUSD', 'ETH/BUSD']"

Sorry if I didn't follow the issue stucture

**Comment by nas-** [comment]
> 
> I'm using binance in dry_run
> This is the error I'm getting "freqtrade | freqtrade.exceptions.OperationalException: Stake-currency 'BTC' not compatible with pair-whitelist. Please remove the following pairs: ['BTC/BUSD', 'ETH/BUSD']"
> 
> Sorry if I didn't follow the issue stucture

You have to change your stake currency from BTC to BUSD in the config file.

**Comment by xmatthias** [comment]
First of all - The issue template needs to be filled out, also for issues, relevant log-messages should be added.

Your's is however not a real bug, but simply a question.



Freqtrade works based on a base currency, set via stake_currency.

If your stake-currency is 'BTC' - then you'r pairs will be "ETH/BTC" / "XRP/BTC" / ...
For 'BTC/BUSD', 'ETH/BUSD' - you'll want to use BUSD as stake-currency (which for live-mode, also assumes that you've got BUSD in your account.
Basically all pairs this bot can trade will "end" with */BUSD, as stake_currency defines the market you're trading in - which seems to be the BUSD market in your case.
